### DISCUSSION TITLE
Jak se postavíme k přístupnosti dynamických dokumentů?

### DISCUSSION THREAD

**Discussion by michalradacz** [egdilna/metodiky-pristupnost-dokumentu#69] [Q&A] [answered]
Tahle metodika má být primnárně o přístupnosti PDF dokumentů, teda statických textových dokumentů. Ale @vladimirdvorakcz  má potřebu řešit přístupnost i u dynamických dokumentů, tedy u videa a podobných věcí. 

Co s tím?

**Comment by vladimirdvorakcz** [comment]
No, ty dynamické dokumenty také dokážou udělat pěknou neplechu v přístupnosti...

From: Michal Rada ***@***.***>
Sent: Wednesday, February 16, 2022 10:58 AM
To: egdilna/metodiky-pristupnost-dokumentu ***@***.***>
Cc: Vladimír Dvořák ***@***.***>; Mention ***@***.***>
Subject: [egdilna/metodiky-pristupnost-dokumentu] Jak se postavíme k přístupnosti dynamických dokumentů? (Discussion #69)


Tahle metodika má být primnárně o přístupnosti PDF dokumentů, teda statických textových dokumentů. Ale @vladimirdvorakcz<https://github.com/vladimirdvorakcz> má potřebu řešit přístupnost i u dynamických dokumentů, tedy u videa a podobných věcí.

Co s tím?

—
Reply to this email directly, view it on GitHub<https://github.com/egdilna/metodiky-pristupnost-dokumentu/discussions/69>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQIAO3GMP3BOYGCNWHIR77DU3NYLFANCNFSM5ORDBO5A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by michalradacz** [comment, marked as answer]
Done
